pkgs/development/ocaml-modules/cohttp/0.17.1.nix

```diff
@@ -0,0 +1,36 @@
+{stdenv, fetchurl, ocaml, findlib, cmdliner, re, uri, fieldslib, sexplib, conduit,
+ stringext, base64, magic-mime, ounit, alcotest, lwt ? null,
+ async ? null, async_ssl ? null, camlp4}:
+
+let
+  ocaml_version = (builtins.parseDrvName ocaml.name).version;
+in
+
+assert stdenv.lib.versionOlder "4.02" ocaml_version;
+
+stdenv.mkDerivation {
+  name = "ocaml-cohttp-0.17.1";
+
+  src = fetchurl {
+    url = https://github.com/mirage/ocaml-cohttp/archive/v0.17.1.tar.gz;
+    sha256 = "fb124fb2fb5ff2e74559bf380627f6a537e208c1518ddcb01f0d37b62b55f673";
+  };
+
+  buildInputs = [ocaml findlib alcotest];
+  propagatedBuildInputs = [camlp4 cmdliner re uri fieldslib sexplib sexplib
+                           conduit stringext base64 magic-mime ounit async
```

Comment:
There is one extra `sexplib` trailing. In fact no occurrence is needed at all, since it’s propagated by, e.g., `conduit`.